crates/router/src/core/payments/transformers.rs

```diff
@@ -1694,6 +1693,8 @@ impl<F: Clone> TryFrom<PaymentAdditionalData<'_, F>> for types::SetupMandateRequ
                     | Some(RequestIncrementalAuthorization::Default)
             ),
             metadata: payment_data.payment_intent.metadata.clone().map(Into::into),
+            minor_amount: Some(amount),
+            amount: Some(amount.get_amount_as_i64()), //need to change once we move to connector module
```

Comment:
You shouldn't migrate existing code, instead add your code `minor_amount` under `amount` field